src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.request;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderRequestDto {
```

Comment:
where do you use it?
![image](https://user-images.githubusercontent.com/20326858/224973738-c13b7478-6ee2-4980-b767-399e427a5529.png)